.github/workflows/main.yml

```diff
@@ -0,0 +1,42 @@
+name: Automated Tests
+
+on: [push, pull_request]
+
+jobs:
+  test:
+    strategy:
+      matrix:
+        python: [3.6, 3.8, 3.9]
+        platform: [ubuntu-latest, macos-latest, windows-latest]
```

Comment:
This needs to be revisited as it kills many platforms.

Reply:
Indeed, it drops support for Python 3.7 and PyPy. It also might drop support beta releases. I wouldn't mind re-introducing pypy support as long as it can be done without overly-complicating the recipe. I'll experiment with that. I omitted Python 3.7 in order to limit the redundant computation required and based on the assumption that if it works on Python 3.6 and 3.8, it almost certainly also works on 3.7.

Reply:
Looks like it's possible to simply add pypy3 to the list of python versions, so I'll go with that for now (176622446).

Reply:
Yep, and they've recently made it possible to separately request pypy3.6 and pypy3.7 which may be useful too.